build.cake

```diff
@@ -405,16 +406,37 @@ Task("Restore")
     RunTool(env.DotNetCommand, "restore OmniSharp.sln", env.WorkingDirectory)
         .ExceptionOnError("Failed to restore projects in OmniSharp.sln.");
 
+    // Restore test assets
```

Comment:
I kind of wish we just used CSX with some bootstrapping code to include a script runner. @filipw has done fantastic work on providing good tooling support for CSX in VS Code and it would give us an opportunity to dogfood it.